From 98b8fb8b1e8baf5573adf31aca9d64aab0a86e67 Mon Sep 17 00:00:00 2001
From: Hans van Kranenburg <hans@knorrie.org>
Date: Fri, 7 Dec 2018 13:11:44 +0100
Subject: [PATCH 16/20] grub-module-verifier: Ignore all_video for xen_pvh

This solves the build failing with "Error: no symbol table and no
.moddeps section"

Also see:
- 6371e9c10433578bb236a8284ddb9ce9e201eb59
- https://savannah.gnu.org/bugs/?49012

Signed-off-by: Hans van Kranenburg <hans@knorrie.org>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
Tested-by: Hans van Kranenburg <hans@knorrie.org>
(cherry picked from commit a40b219e269ac407c10a569ce6b0c5d7f419d127)
---
 util/grub-module-verifier.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/util/grub-module-verifier.c b/util/grub-module-verifier.c
index 9179285a5..968eb135d 100644
--- a/util/grub-module-verifier.c
+++ b/util/grub-module-verifier.c
@@ -128,6 +128,7 @@ struct platform_whitelist {
 
 static struct platform_whitelist whitelists[] = {
   {"i386", "xen", (const char *[]) {"all_video", 0}},
+  {"i386", "xen_pvh", (const char *[]) {"all_video", 0}},
   {"x86_64", "xen", (const char *[]) {"all_video", 0}},
   {"sparc64", "ieee1275", (const char *[]) {"all_video", 0}},
 
-- 
2.21.0

